storage/engine/rocksdb/db.cc

```diff
@@ -115,6 +120,15 @@ DBStatus ToDBStatus(const rocksdb::Status& status) {
   return ToDBString(status.ToString());
 }
 
+DBStatus FmtStatus(const char *fmt, ...) {
```

Comment:
I'm more concerned about returning a pointer to freed memory than a leak (I don't see any practical way to test for the absence of a leak). I'd just like to see a test where MVCCStats returns an error to make sure we execute FmtStatus at least once.